core/src/main/java/cucumber/api/event/TestCaseStarted.java

```diff
@@ -4,10 +4,15 @@
 
 public final class TestCaseStarted extends TestCaseEvent {
     public final TestCase testCase;
+    private final long timeStampMillis; 
 
-    public TestCaseStarted(Long timeStamp, TestCase testCase) {
+    public TestCaseStarted(Long timeStamp, Long timeStampMillis, TestCase testCase) {
```

Comment:
Please add a deprecated `public TestCaseStarted(Long timeStamp, TestCase testCase)` constructor that calls `this(timeStamp, 0, testCase)`. The `TestCaseStarted` event is part of the public API so we should deprecate the constructor before removing it.

Reply:
Thanks that's useful knowledge :) Updated.